scripts/scraper/venv/lib/python3.12/site-packages/_pytest/_argcomplete.py

```
@@ -0,0 +1,117 @@
+import os
+import sys
+from typing import Any
+
+
+class FastFilesCompleter:
+    """Fast file completer class."""
+
+    def __init__(self, directories: bool = True) -> None:
+        self.directories = directories
+
+    def __call__(self, prefix: str, **kwargs: Any) -> list[str]:
```

Ruff (0.14.1)

Unused method argument: `kwargs`

(ARG002)